internal/exec/helmfile.go

```diff
@@ -86,7 +86,7 @@ func ExecuteHelmfile(cmd *cobra.Command, args []string) error {
 	clusterName := c.ReplaceContextTokens(context, c.Config.Components.Helmfile.ClusterNamePattern)
 	color.Cyan(fmt.Sprintf("Downloading kubeconfig from the cluster '%s' and saving it to %s\n\n", clusterName, kubeconfigPath))
 
-	err = execCommand("aws",
+	err = ExecuteShellCommand("aws",
```

Comment:
Can we easily allow `--role-arn` for the `helmfile` command?

Reply:
let's not do it in this PR